src/main/java/nextstep/subway/domain/path/fee/AgeType.java

```diff
@@ -0,0 +1,45 @@
+package nextstep.subway.domain.path.fee;
+
+import nextstep.auth.application.UserDetails;
+import nextstep.member.domain.Member;
+
+public enum AgeType {
+    YOUTH {
+        @Override
+        public Fare calculate(final Fare fare) {
+            return fare.minus(Fare.deduction()).discount(20);
+        }
+    },
+    CHILDREN {
+        @Override
+        public Fare calculate(final Fare fare) {
+            return fare.minus(Fare.deduction()).discount(50);
+        }
+    },
```

Comment:
Function을 활용해 람다식으로 선언하면 어떨까요?
```suggestion
    CHILDREN (fare -> fare.minus(Fare.deduction()).discount(50)),
```